tests/test_search_open.py

```
@@ -16,3 +23,110 @@ def test_find_anchor_line(tmp_path: Path) -> None:
+    target.write_text("<!doctype html><html></html>", encoding="utf-8")
+    ui = _RecordingUI()
+    hit = _hit_for_path(target)
+
+    opened: list[str] = []
+
+    monkeypatch.setattr("polylogue.cli.search_cli.webbrowser.open", lambda url: opened.append(url) or True)
+    monkeypatch.setattr("polylogue.cli.search_cli.open_in_editor", lambda *_a, **_k: pytest.fail("open_in_editor should not be called"))
+
+    _open_single_search_hit(ui, hit)
+
+    assert opened and opened[0].endswith("#msg-2")
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)